src/api/modules/localist.ts

```diff
@@ -16,13 +16,14 @@ const CACHE_SEC = parseInt(config.get('localist.cacheEndpointSec'), 10);
  */
 export const getEvents = async (query: any): Promise<object[]> => {
   try {
-    const urlParams = querystring.stringify(query);
+    // Ensure that a today `YYYY-MM-DD` is included in the query parameters for localist
+    const urlParams = querystring.stringify({
+      ...query,
+      start: new Date().toISOString().substr(0, 10)
+    });
     const url = `${LOCALIST_BASE_URL}/events?${urlParams}`;
     const data = await cache.get(url, { json: true }, true, { key: url, ttlSeconds: CACHE_SEC });
```

Comment:
Is it _possible_ for data to come back and NOT have a .url defined? If so we should probably code some exception handling in there for that scenario. If not an issue, nevermind!